lib/api-key.ts

```diff
@@ -89,15 +89,16 @@ export class APIKey implements Token {
 	public isExpired = (): boolean => false;
 
 	/**
-	 * @member needs2FA
-	 * @summary Check if the given token requires 2FA
+	 * @member get2FAStatus
+	 * @summary Gets whether passing a 2FA challenge is pending, passed or not required.
 	 * @function
 	 * @public
 	 *
-	 * @returns {boolean}
+	 * @returns {'not_required'|'pending'|'passed'}
 	 *
 	 * @example
-	 * console.log(token.needs2FA());
+	 * console.log(token.get2FAStatus())
 	 */
-	public needs2FA = (): boolean => false;
+	public get2FAStatus = (): 'not_required' | 'pending' | 'passed' =>
```

Comment:
Trying to find a word that would make sense in both JWT and API Key cases and ended up with `not_required`. Suggestions welcome.

Reply:
I agree this get2FAStatus is the way to go, I am not sure this won't break things if we remove a function in a minor?

Reply:
`not_required` sounds fine to me